### PR TITLE
[FIX] delivery: impossible to open transfert view

### PR DESCRIPTION
**Pull request by fmdl**
Description of the issue/feature this PR addresses:
- Install delivery
- Create 80 pickings to transfert, and each picking have 1000 stock.move.line
--> Issue the time to load the page is very very long

Because `shipping_weight` is a pure python unstored computed field.

@amoyaux @Whenrow 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Whenrow** [comment]
@fmdl We should rather try to improve the compute method instead of removing the field from the view. We will maybe store the value in master also.

**Comment by fmdl** [comment]
@Whenrow it is updated

**Comment by fmdl** [comment]
ping @Whenrow

File: addons/delivery/models/stock_picking.py [hunk 1]
```diff
@@ -44,15 +45,16 @@ def _compute_weight_uom_name(self):
...
-        for package in self:
-            packs = set()
-            for move_line in package.move_line_ids:
-                if move_line.result_package_id:
-                    packs.add(move_line.result_package_id.id)
-            package.package_ids = list(packs)
+        picking_map = defaultdict(list)
+        data = self.env['stock.move.line'].read_group(
+            [('picking_id', 'in', self.ids), ('result_package_id','!=',False)], 
+            ['picking_id', 'result_package_id'], ['picking_id', 'result_package_id'], lazy=False)
```
Comment (Whenrow) [inline addons/delivery/models/stock_picking.py:53]: you can make the group directly in sql with array_agg
```suggestion
            ['picking_id', 'result_package_id'], ['picking_id', 'result_package_id:array_agg'], lazy=False)
```
Also could you give some figures about the time saved by this patch in the PR description ? Thanks a lot

**Pull request closed without merging**